### PR TITLE
[REL] website_sale_product_descripion

### PR DESCRIPTION
**Pull request by ValentinoDefelice**
[FIX] Update models and views file names
[FIX] Update README

### CODE REVIEW

**Review by zaoral — APPROVED** [review]
+1

@ValentinoDefelice 
Please only need to fix a pylint error `website_sale_product_description/models/product_template.py:11:1: E303 too many blank lines (3)` https://travis-ci.org/ingadhoc/website/jobs/394054614#L828

File: website_sale_product_description/README.rst [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
 To configure this module, you need to:
 
+#. Nothing to configure.
 
 Usage
 =====
 
 To use this module, you need to:
 
+#. Go to a Product, and you will see in 'Sale' page a 'Description for Website' title with a Description field.
```
Comment (zaoral) [inline website_sale_product_description/README.rst]: > +#. Go to a Product, and you will see in 'Sale' page a 'Description for Website' title with a Description field.

change with 

> +#. Go to a Product, and you will see in 'Notes' page a 'Description for Website' text field

File: website_sale_product_description/models/product_template.py [hunk 2]
```diff
@@ -7,7 +7,6 @@
 _logger = logging.getLogger(__name__)
```
Comment (zaoral) [inline website_sale_product_description/models/product_template.py]: remove both _logger variable and import logging. this two are not necessary here

File: website_sale_product_description/models/product_template.py [hunk 3]
```diff
@@ -7,7 +7,6 @@
...
 
 
-# configAuthorization.set('PRISMA f3d8b72c94ab4a06be2ef7c95490f7d3')
```
Comment (zaoral) [inline website_sale_product_description/models/product_template.py]: we have a pylint error of two many empty lines, please fix this
https://travis-ci.org/ingadhoc/website/jobs/394054614#L828

File: website_sale_product_description/models/product_template.py [hunk 4]
```diff
@@ -7,7 +7,6 @@
...
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
```
Comment (zaoral) [inline website_sale_product_description/models/product_template.py]: We have already a Sale Description field in the sale order. please update `description_website` field string to "Description for Website"

File: website_sale_product_description/views/product_template_views.xml [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
-
     <record model="ir.ui.view" id="product_template_form_view">
         <field name="name">product.template.website.sale.product.description.form</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_form_view"/>
         <field name="arch" type="xml">
             <group name="email_template_and_project" position="after">
-                <separator string="Description for Website"/>
-                <field name="description_website" placeholder="note to be displayed on website..."/>
+                <group>
```
Comment (zaoral) [inline website_sale_product_description/views/product_template_views.xml]: * Currently it looks like this

  image

* Please make it look like this one 

  image

   - dont use separator, use group string instead
   - all the descriptions of the product are now in the Notes page, please move it to this section
   - Dont show the field label.

```xml
<group name="description" position="inside">
    <group string="Description for Website">
        <field name="description_website" nolabel="1" placeholder="Note to be displayed on e-commerce..."/>
    </group>
</group>
```

**Review by zaoral — CHANGES_REQUESTED** [review]

**Review by nicomacr — DISMISSED** [review]
Hi @ValentinoDefelice for me it's OK the PR

**Review by zaoral — DISMISSED** [review]

**Pull request merged**